workers/unity/Packages/io.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -247,6 +248,26 @@ private void OnGUI()
                     EditorGUILayout.LabelField("Project Name", projectName);
                 }
 
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    using (new EditorGUI.DisabledScope(manager.IsActive))
+                    {
+                        var buttonIcon = new GUIContent(EditorGUIUtility.IconContent(BuiltInEditIcon))
```

Comment:
Is it making this every frame?

Reply:
Hmm good question, to be honest - I cargo-culted this from the other buttons in this window. I can try making them readonly/static and seeing what happens